querybook/webapp/components/DataTableView/DataTableView.tsx

```diff
@@ -168,6 +192,9 @@ class DataTableViewComponent extends React.PureComponent<
                 onTabSelected={this.onTabSelected}
                 updateDataTableDescription={this.updateDataTableDescription}
                 onExampleFilter={this.handleExampleFilter}
+                onEditMetadata={this.getOnEditMetadata(
```

Comment:
onBeforeEditDescription or onEditDescriptionRedirect?

Reply:
I think it's not "before" actually, but it's "on editing" when clicking, although it will go to another site for editing. 
i'll rename is as `onEditTableDescription`